src/cosl/coordinated_workers/nginx.py

```diff
@@ -205,3 +206,15 @@ def layer(self) -> pebble.Layer:
                 },
             }
         )
+
+
+def is_ipv6_enabled() -> bool:
+    """Check if IPv6 is enabled on the container's network interfaces."""
+    exit_code, output = subprocess.getstatusoutput("ip -6 address show")
```

Comment:
I'd use `run` instead; it has a bunch of handy args for much of what you're manually doing below:

```
def is_ipv6_enabled():
    try:
        out = subprocess.run(["ip", "-6", "address", "show"], check=True, capture_output=True, text=True)
    except:
        return False
    return bool(out)
```